feeds/wlan-ap/opensync/src/platform/openwrt/src/lib/target/src/stats.c

```diff
@@ -372,7 +372,10 @@ bool target_stats_device_temp_get(radio_entry_t *radio_cfg, dpp_device_temp_t *t
 
 	fclose(fp);
 	temp_entry->type  = radio_cfg->type;
-	temp_entry->value = temperature / 1000;
+	if(temperature > 125)
+		temp_entry->value = temperature / 1000;
+	else
+		temp_entry->value = temperature;
```

Comment:
This seems very arbitrary.  If there is a scaling difference between the two methods of obtaining temperature data, then make the scaling scaling change at the point of retrieval.  This way you will know explicitly which scale applies.

Reply:
done